src/models/response.ts

```diff
@@ -15,11 +15,11 @@ export class Response {
      * Returns "true" if this response is meaningful from documentation prospective.
      */
     public isMeaningful(): boolean {
-        return !!this.description || this.representations.some(x => !!x.typeName || !!x.example);
+        return !!this.description || this.representations.some(x => !!x.typeName || x.examples.length > 0);
```

Comment:
do we need to be aware if there are no examples? - x.examples?

Reply:
Yep, added checks.